.github/workflows/pull-request-workflow.yml

```diff
@@ -0,0 +1,16 @@
+name: Pull request workflow
+
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
I think it might be nicer to have this on a `prototype` branch, like you'd have `gh-pages`. Then that branch could also be setup for the heroku deploy. Thoughts?

Reply:
Mostly where I've seen a `gh-pages` branch it's because you need to commit build artifacts to GitHub for pages to display. But Heroku lets us compile the prototype on their servers, so we can just build off master and any prototype changes go through the same test/release process as changes to code.

I'm hoping that will make us treat the prototype as a first class citizen rather than a bunch of files we let get out of date.